packages/grafana-ui/src/components/uPlot/plugins/TooltipPlugin.tsx

```diff
@@ -263,8 +264,12 @@ export const TooltipPlugin = ({
     tooltip = renderTooltip(otherProps.data, focusedSeriesIdx, focusedPointIdx);
   }
 
+  const activeTooltipClass = css`
+    z-index: ${theme.zIndex.portal + 1} !important;
+  `;
+
```

Comment:
you can probably avoid re-generating this static class on every tooltip movement and extract it out of this re-rendering component by using the getStyles/useStyles2 pattern. see what we're doing here:

https://github.com/grafana/grafana/pull/70872/files#diff-8b78102d87c552bbe06c1e3887b7bb6c1bf2742c92d94431433daf9e3aaa5899